Lib/test/test_str.py

```diff
@@ -1747,6 +1747,18 @@ def test_constructor_keyword_args(self):
         self.assertEqual(str(b'foo', errors='strict'), 'foo')  # not "b'foo'"
         self.assertEqual(str(object=b'foo', errors='strict'), 'foo')
 
+    def test_constructor_errors(self):
```

Comment:
Actually, constructor errors are tested in `test_str_invalid_call`. There is also one test in `test_constructor`: `self.assertRaises(TypeError, str, 42, 42, 42)`. It would be nice to unify them. I would also add tests with `encoding` or `errors` passed as positional and keyword arguments (simultaneously).